backend/src/channel-invited/channel-invited.resolver.ts

```diff
@@ -0,0 +1,52 @@
+import { Args, Mutation, Resolver, Query } from '@nestjs/graphql'
+import { ChannelInvited } from './entities/channel-invited.entity'
+import { ChannelInvitedService } from './channel-invited.service'
+import { CreateChannelInvitedInput } from './dto/create-channel-invited.input'
+import { ValidationPipe } from '@nestjs/common'
+
+@Resolver(() => ChannelInvited)
+export class ChannelInvitedResolver {
+  constructor(private readonly channelInvitedServie: ChannelInvitedService) {}
+
+  //**************************************************//
+  //  MUTATION
+  //**************************************************//
+  @Mutation(() => ChannelInvited)
+  async createChannelInvited(
+    @Args('data', { type: () => CreateChannelInvitedInput }, ValidationPipe)
+    createChannelInvitedDto: CreateChannelInvitedInput
+  ): Promise<ChannelInvited> {
+    return this.channelInvitedServie.create(createChannelInvitedDto)
+  }
+
+  @Mutation(() => ChannelInvited)
+  async deleteChannelInvited(
+    @Args('userId', { type: () => String }, ValidationPipe)
```

Comment:
ValidationPipe is important only for dtos, you can't place a Validation Pipe on string field, since you don't have an update and only one create function, you'll user one single validationPipe on the create function and that's it